fdbserver/workloads/AuthzSecurity.actor.cpp

```diff
@@ -176,7 +191,12 @@ struct AuthzSecurityWorkload : TestWorkload {
 			           crossTenantBGLocNegative.getValue() > 0 && crossTenantBGReqPositive.getValue() > 0 &&
 			           crossTenantBGReqNegative.getValue() > 0 && crossTenantBGReadPositive.getValue() > 0 &&
 			           crossTenantBGReadNegative.getValue() > 0 && crossTenantGetGranulesPositive.getValue() > 0 &&
-			           crossTenantGetGranulesNegative.getValue() > 0;
+			           crossTenantGetGranulesNegative.getValue() > 0 && blobbifyNegative.getValue() > 0;
```

Comment:
It doesn't look like `blobbifyNegative` belongs here (it's being accounted for in `checkBlobManagement` conditional block).

Reply:
Interesting, it's strange that the tests passed with this check since that should have caused the check phase to return false in cases where blob management was disabled

Reply:
To be strict, `&=` is a bitwise-AND operation whereas we're working with logical-ANDs.
Just to be sure, could you try `success = success && ...` instead?

Reply:
I checked with that too, the ASSERT(success) still fails